RedHat-UX/red-hat-design-system
# docs(table): update rh-table best practices

## What I did

1. Switched `<rh-table>`'s "Best practices" section to the do/don't format


## Testing Instructions

1. Check ["Best practices" in DP](https://deploy-preview-2061--red-hat-design-system.netlify.app/elements/table/guidelines/#best-practices).

## Notes to Reviewers
